sklearn/ensemble/_gb_losses.py

```diff
@@ -527,6 +527,8 @@ def _raw_prediction_to_proba(self, raw_predictions):
         -------
         probas : 2d array, shape (n_samples, K)
             The predicted probabilities.
+
+        .. versionadded:: 0.20.3
```

Comment:
What is the reason you chose 0.20.3 over 0.20 ?

Reply:
I can change it to 0.20. I have no idea why 20.3 was chosen.

Reply:
please do :)